bsp/resources/messages/ScalaBspBundle.properties

```diff
@@ -10,10 +10,10 @@ bsp.widget.connections.on=on
 bsp.widget.connections.off=off
 
 ### org/jetbrains/bsp/data/dataObjects.scala
-bsp.metadata.error.project.info=BspMetadata.get: Could not get project info of project '{0}'
```

Comment:
You may use double apostrophes to escape them.

Reply:
Thanks! I'll do it later